### PR TITLE
Start Steve Home Town File

### PR DESCRIPTION
**Pull request by SteveBaston**
### Description of Change

- Creating a file for my home town

### Test results

- NO

### Who should know about the change?

/cc: @FloSeguin

### CODE REVIEW

File: Stevehometown.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Kenilworth
+
+#Castle
```
Comment (SteveBaston) [inline Stevehometown.md:3]: how old is the castle?

File: Stevehometown.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+#things to do for children
```
Comment (SteveBaston) [inline Stevehometown.md:5]: swimming pool

File: Stevehometown.md (hunk 1 above)
Comment (al517) [inline Stevehometown.md:3]: where is Steve Home Town?

**Review by FloSeguin — APPROVED** [review]
ok

**Review by FloSeguin — APPROVED** [review]

**Pull request merged**